packages/babel-plugin-transform-function-sent/test/fixtures/generator-kinds/class-method/expected.js

```diff
@@ -0,0 +1,12 @@
+function _skipFirstGeneratorNext(fn) { return function () { var it = fn.apply(this, arguments); it.next(); return it; }; }
+
+class Foo {
+  gen() {
+    return _skipFirstGeneratorNext(function* () {
```

Comment:
Doing so would break code like this: https://github.com/babel/babel/blob/7.0/packages/babel-plugin-transform-async-to-generator/test/fixtures/async-to-generator/object-method-with-super/actual.js

(I'm linking a test of async-to-generator instead of function-sent because it is basically a test for helper-wrap-function, which is the plugin which would host that call)